include/correction/QnTools/CorrectionCuts.hpp

```diff
@@ -231,20 +231,31 @@ class CorrectionCuts {
 
 namespace CallBacks {
```

Comment:
maybe we can remove the call back an directly create the cut. It will simplify the solution a bit.